registry/ccle-wgs.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "ccle-wgs",
+    "description": "Cancer Cell Line Encyclopedia whole genome sequencing",
+    "target": "ccle/wgs",
```

Comment:
Nit: We've updated the format recently, so that `target` is no longer used (see https://github.com/bigdatagenomics/eggo/blob/master/docs/spec.md), so this line can be omitted. (I still need to update the other files in the registry).